contracts/Marketplace.sol

```diff
@@ -178,6 +182,9 @@ contract Marketplace is Collateral, Proofs {
   }
 
   function proofEnd(bytes32 slotId) public view returns (uint256) {
+    if (!_slotAcceptsProofs(slotId)) {
+      return block.timestamp - 1;
+    }
     return _end(slotId);
```

Comment:
If `_end(slotId)` is in the past, I would return that, instead of `block.timestamp - 1`.

Reply:
Updated.